src/main/scala/mesosphere/marathon/event/Events.scala

```diff
@@ -206,6 +207,16 @@ case class AppTerminatedEvent(
   eventType: String = "app_terminated_event",
   timestamp: String = Timestamp.now().toString) extends MarathonEvent
 
+case class UnhealthyTaskKillEvent(
```

Comment:
I added a note in 
* docs/docs/event-bus.md
* docs/docs/rest-api/mesosphere/marathon/api/v2/AppsResource_create.md